QUARKUS-715.md

```diff
@@ -0,0 +1,55 @@
+# QUARKUS-715 Support for multiple Hibernate ORM persistence units
+
+This document describes a high-level testing overview of [QUARKUS-715 Support for multiple Hibernate ORM persistence units](https://issues.redhat.com/browse/QUARKUS-715).
+
+## Scope of the testing
+
+### Component testing
```

Comment:
This was really useful information! As Hibernate is a third party component, I would move this later in this test plan as an extra bits or something.